packages/confirmation-modal/src/ConfirmationModal.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useMemo, useState } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@leafygreen-ui/emotion';
+import Button, { Variant } from '@leafygreen-ui/button';
+import Modal from '@leafygreen-ui/modal';
+import { uiColors } from '@leafygreen-ui/palette';
+import TextInput from '@leafygreen-ui/text-input';
+
+const titleStyle = css`
+  color: ${uiColors.gray.dark2};
+  font-size: 24px;
+  font-weight: bold;
+  line-height: 25px;
+
+  margin-bottom: 10px;
+`;
+
+const baseModalStyle = css`
+  width: 600px;
+  padding: initial;
+`;
+
+const contentStyle = css`
+  color: ${uiColors.gray.dark1};
+  font-family: Akzidenz, ‘Helvetica Neue’, Helvetica, Arial, sans-serif;
+  font-size: 16px;
+  letter-spacing: 0;
+  line-height: 24px;
+
+  padding: 36px;
+`;
+
+const textEntryInputStyle = css`
+  width: 300px;
+  padding-top: 14px;
+`;
+
+const footerStyle = css`
+  &:before {
+    content: '';
+    display: block;
+    position: relative;
+    width: 100%;
+    border-top: 1px solid ${uiColors.gray.light2};
+  }
+`;
+
+const footerContentStyle = css`
+  display: flex;
+  justify-content: right;
+  flex-direction: row-reverse;
```

Comment:
I tend to find this harder to understand when reading the JSX for the component. Is there any reason we need row-reverse here rather than just ordering the DOM appropriately?

Reply:
my main reasoning was to influence the tabbing order so that the primary action comes first and I think logically makes sense since the buttons are right-aligned.

Reply:
Got it – makes sense. I'm down with not manipulating tabIndex 👍